content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -145,7 +145,7 @@ <h3>View the app</h3>
                 <p><b><code>export POD_NAME=$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')</code></b><br />
                    <b><code>echo Name of the Pod: $POD_NAME</code></b></p>
                 <p>You can access the Pod through the proxied API, by running:</p>
-                <p><b><code>curl http://localhost:8001/api/v1/namespaces/default/pods/$POD_NAME/</code></b></p>
+                <p><b><code>curl http://localhost:8001/api/v1/namespaces/default/pods/$POD_NAME/proxy/</code></b></p>
```

Comment:
```suggestion
                <p>You can access the Pod object through the proxied API, by running:</p>
                <p><b><code>curl http://localhost:8001/api/v1/namespaces/default/pods/$POD_NAME/</code></b></p>
</b></p>
```

A diagram would help here. What we're doing is not the equivalent of `kubectl proxy` (scenario #<span/>1 in [Proxies in Kubernetes](https://kubernetes.io/docs/concepts/cluster-administration/proxies/)), it's actually scenario #<span/>2).

Reply:
OK, that makes sense to me. I think your proposed change would have clarified it to me. Thanks!

I think the text after the example, "in order for the new Deployment to be accessible without using the proxy, a Service is required," made me think we should be accessing the pod's service itself (since that's what a Service would do). Maybe the text should be changed to say something like "in order for the new Deployment to be reliably accessible, a Service is required" (or "generally accessible" or something like that). Or otherwise maybe first explain that one can access a pod's application through the proxy with `/$POD_NAME/proxy/` before saying a Service can be used to access it "without using the proxy."